benches/reader.rs

```diff
@@ -0,0 +1,86 @@
+#![feature(test)]
+extern crate grcov;
+extern crate test;
+
+use grcov::{GcovReaderBuf, GCNO};
+use std::path::PathBuf;
+use test::{black_box, Bencher};
+
+#[bench]
+fn bench_reader_gcno(b: &mut Bencher) {
+    let mut gcno = GCNO::new();
+    b.iter(|| {
+        let file = GcovReaderBuf::from("test/llvm/reader.gcno");
+        black_box(gcno.read(file).unwrap());
+    });
+}
+
+#[bench]
+fn bench_reader_gcno_gcda(b: &mut Bencher) {
+    let mut gcno = GCNO::new();
+    gcno.read(GcovReaderBuf::from("test/llvm/reader.gcno"))
+        .unwrap();
+
+    b.iter(|| {
+        let file = GcovReaderBuf::from("test/llvm/reader.gcda");
+        black_box(gcno.read_gcda(file).unwrap());
+    });
+}
+
+#[bench]
+fn bench_reader_gcno_counter(b: &mut Bencher) {
```

Comment:
This is benchmarking the dump function, so you can call it bench_reader_gcno_dump